### PR TITLE
added negative direction speed

### PR DESCRIPTION
**Pull request by nkristina**

### CODE REVIEW

File: source/dagger/gameplay/ping_pong/player_scores.cpp [hunk 1]
```diff
@@ -50,6 +50,7 @@ void PlayerScoresSystem::Run()
     if (ballOnField == 0)
     {
         // TODO: set speed to be random in both directions
-        CreatePingPongBall(s_TileSize, ColorRGBA(1, 1, 1, 1), { rand()%10 + 4,rand()%10 + 4,0 },   { 0,rand()%(s_FieldHeight / 2),0 });
+		float x_speed = rand() % 10 - 3.12;
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/player_scores.cpp]: x_speed can be generated with 0, so the game will freeze, please try to fix it

**Review by DPrivalov — CHANGES_REQUESTED** [review]

File: source/dagger/gameplay/ping_pong/player_scores.cpp (hunk 1 above)
Comment (UbiBelETF) [reply source/dagger/gameplay/ping_pong/player_scores.cpp]: Also, `3.12` is a double and not a float (like `3.12f` is). Watch out for type sizes and implicit conversions...

**Review by milosmilosevic13 — APPROVED** [review]

File: source/dagger/gameplay/ping_pong/player_scores.h [hunk 2]
```diff
@@ -28,5 +28,12 @@ namespace ping_pong
             s_FieldHeight = height;
             s_TileSize = tileSize;
         }
+
+		float GetRandomSpeed()
+		{
+			float x_speed = rand() % 10 - 5.0f;
+			x_speed = x_speed ? x_speed : x_speed + 5.0f;
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/player_scores.h]: you are trying to compare with 0 here?
it's bad practice to compare float with 0 like that. You need to use > < operators instead.

**Review by DPrivalov — CHANGES_REQUESTED** [review]

**Review by DPrivalov — APPROVED** [review]
ball still might be too slow, but it's much better already

**Pull request merged**